sample/build.gradle

```diff
@@ -34,6 +34,10 @@ android {
   sourceSets {
     main.java.srcDirs += 'src/main/kotlin'
   }
+
+  lintOptions {
+    abortOnError true
```

Comment:
This is the default one. So no need to configure it

Reply:
Fixed in 26576f1577197783703616ad4123e5be147b7ad3